### ISSUE TITLE
Forum web components

### ISSUE THREAD

**Issue by wabscale** [AnubisLMS/Anubis#460]
We can start simple, and not wire things up. This means designing components for:

- List of posts
- Writing a post (ie: textbox + live markdown + post button). Should this be a [dialog](https://mui.com/material-ui/react-dialog/)?
- Viewing a post (ie: translate content to markdown)
- Writing a comment (ie: textbox + post button) 
- Comment thread under post (ie: text content + approved badge for approved answers)

We may want to also include basic logic here for automatically sorting posts by category

**Comment by dolf321** [comment]
I'll take this @wabscale

**Comment by dolf321** [comment]
Yep just deployed needs a lot of work, ill take it! Should probably also include search functionality in this issue or elsewhere

**Comment by GusSand** [comment]
I'm gonna create a new item for search
